character-sfx.js

```diff
@@ -342,8 +351,49 @@ class CharacterSfx {
   
     };
     _handleNarutoRun();
-    
 
+    // combo
+    const dispatchComboSoundEvent = (soundIndex) =>{
+      this.dispatchEvent(new MessageEvent('meleewhoosh', {
+        data: {
+          index: soundIndex
+        },
+      }));
+    }
+    const _handleCombo = () => {
+      if (this.player.hasAction('use') && this.player.getAction('use').behavior === 'sword' && this.player.getAction('use').animationCombo) {

```

Comment:
Why look it up 3 times per `if`?

Reply:
[clean code](https://github.com/webaverse/app/pull/3674/commits/9954dfb8d6a95d9adbf0fc9cb796d8e5a7e9b722)